### DISCUSSION TITLE
Correct streamflow data by area in L11 fAcc file

### DISCUSSION THREAD

**Discussion by ViniciusBMoreira** [mhm-ufz/mHM#33] [Q&A] [answered]
Hey  again guys!

Since i managed to run the model, i have read many articles done by you to calibrate and set good initial conditions in the model.

But after noticing that one of my stations had bad indexes during calibration, I saw that level l11 did not position it correctly, as shown in the figure:

(the clear area represents high values ​​of flow accumulation, where my station should be)

![image](https://user-images.githubusercontent.com/61054098/124525526-fd289580-ddd5-11eb-9ec1-69e71e399a0c.png)

But I resolved this issue without any problems. But from there I started to analyze the real area values ​​and the ones obtained by the fAcc file and realized that the results were up to 25% different. So I corrected the flow values ​​based on that area. However, now I am afraid and wondering if the model itself does not have this calculation or something like this!

Ty  for the always Help!

Cheers from Brazil!

Bryan

**Comment by rohinikumar82** [comment, marked as answer]
Dear @BryanSouzaM,
This is somehow a common issue - that arises due to difference in data-sources/processing steps (one that is used to determine terrain characteristics and one where gauges information are provided.  This the user  need to take into account during the data processing steps  Specific to this, I suggest you can move the gauging station info (in idgauges.asc file) to the nearest stream (here you can use the facc.asc map) in some GIS softwares. Where to move would depend for example the criteria of minimum mismatch between gauged catchment (upstream) area and the one that you get from the facc map.   And then export the new adjusted idgauges.asc file and use it for your set-up. 
 
Best,
Rohini

**Reply by ViniciusBMoreira** [reply]
Perfect @rohinikumar82, that's what I did. But there is a difference between the real area and the area obtained from the fAcc file. Is it acceptable to correct inlet flow data based on this area difference?

Bryan

**Reply by rohinikumar82** [reply]
I guess you mean not the flow (Q) data - but the flow location (idgauges.asc) based on facc map using the criteria of minimal mismatch between observed area (got specific gauge) and one you get based on facc map... this is quite ok!
-Rohini

**Reply by ViniciusBMoreira** [reply]
Perfect.

thanks!
Bryan
